lib/web_ui/lib/src/engine/compositor/canvas.dart

```diff
@@ -185,7 +186,8 @@ class SkCanvas {
     skCanvas.callMethod('drawPicture', <js.JsObject>[skPicture.skPicture]);
   }
 
-  void drawPoints(SkPaint paint, ui.PointMode pointMode, Float32List points) {
+  void drawPoints(SkPaint paint, ui.PointMode pointMode,
+      js.JsArray<js.JsArray<double>> points) {
```

Comment:
Can you please file an issue and leave a TODO to turn this into a single flat typed array?

Reply:
Done.